.github/workflows/prauto.yml

```
@@ -0,0 +1,67 @@
+name: PR Automation
+on:
+  pull_request:
+    types: [ready_for_review, opened, reopened]
+jobs:
+  pr-auto:
+    runs-on: ubuntu-latest
+    steps:
+      - name: lint pr
+        env:
+          REPO: "cli/cli"
+          NEEDSREVIEWCOL: "MDEzOlByb2plY3RDb2x1bW43MTEwMTI4"
+          PRID: ${{ github.event.pull_request.node_id }}
+          PRBODY: ${{ github.event.pull_request.body }}
+          STAFF: "vilmibm,mislav,samcoe,billygriffin,ampinsk"
+          PRNUM: ${{ github.event.pull_request.number }}
+          PRBASE: ${{ github.event.pull_request.base.ref }}
+          GH_TOKEN: ${{ secrets.AUTOMATION_TOKEN }}
+          PRAUTHOR: ${{ github.event.pull_request.user.login }}
+          # Testing data:
+          #REPO: "vilmibm/testing"
+          #NEEDSREVIEWCOL: "MDEzOlByb2plY3RDb2x1bW4xMTAzNTA3Ng==" # vilmibm/testing "good" column
+        run: |
+          commentPR () {
+            gh -R$REPO pr comment $PRNUM -b "${1}"
+          }
+
+          closePR () {
+            gh -R$REPO pr close $PRNUM
+          }
+
+          addToBoard () {
+            # this exec is an attempt to fix some quote highlighting issues i was having in my editor.
+            bash -c 'gh api graphql -f query="mutation { addProjectCard(input: { projectColumnId: \"$NEEDSREVIEWCOL\", contentId: \"$PRID\" }) { clientMutationId } }"'
```

Passing graphql variables as separate arguments avoids quoting issues:
```suggestion
            gh api graphql -f query='mutation($col:ID!, $pr:ID!) { addProjectCard(input: { projectColumnId: $col, contentId: $pr }) { clientMutationId } }' -f col="$NEEDSREVIEWCOL" -f pr="$PRID"
```